package.json

```diff
@@ -4,15 +4,16 @@
   "description": "JSON RPC server for Shardeum",
   "main": "src/server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
@kenny-io  why was this removed? `npm test` is going to be expected to be here for the github workflow automation